### ISSUE TITLE
Resizing window causes dropdowns to be stuck

### ISSUE THREAD

**Issue by octopusxl** [mrjasonweaver/flexnav#116]
Not sure if it's related to Issue https://github.com/indyplanets/flexnav/issues/62, but resizing the window causes the drop down menus to be stuck. I'm on Chrome 32 and Safari 7 on OS 10.9.1. 

![flexnav-menu](https://f.cloud.github.com/assets/203176/2002715/d1bec85a-8609-11e3-8fde-b078acc0904c.jpeg)

**Comment by ukeboyska** [comment]
yep, this happened to me in chrome, too...it looks pretty good on desktop, otherwise

**Comment by NielsLust** [comment]
Hi,

I've had the same problem, fixed it with a little jQuery (on resize) 

$('.item-with-ul ul').hide();
$('.touch-button.active').removeClass('active');

The problem that i can't fix is after opening a submenu item on small-screens and then resize it back to desktop-size the submenu flickers on mouse enter and sometimes does not show..

Please help :)

Niels

[edit]: Fixed with Issue #119
